core/pom.xml

```diff
@@ -106,7 +106,7 @@
         <dependency>
             <groupId>org.jahia.modules</groupId>
             <artifactId>graphql-dxm-provider</artifactId>
-            <version>2.20.0-SNAPSHOT</version>
+            <version>3.0.0-SNAPSHOT</version>
```

Comment:
Seem's there is old import-package that need some cleanup.
if we bump gql dependency to 3.0.0-SNAPSHOT it's ok because jahia parent is set to 8.2.0.0-SNAPSHOT.

then we can remove those explicit import-package: 
```
org.jahia.modules.graphql.provider.dxm;version="[2.20,4)",
org.jahia.modules.graphql.provider.dxm.admin;version="[2.20,4)",
org.jahia.modules.graphql.provider.dxm.osgi.annotations;version="[2.20,4)"
```